src/js/api/brand.js

```diff
@@ -123,6 +123,22 @@ ripe.Ripe.prototype.getMeshUrl = function(options) {
     return options.url + "?" + this._buildQuery(options.params);
 };
 
+/**
+ * Returns the URL where the font can be retrieved.
+ *
+ * @param {Object} options A map with options, such as:
+ *  - 'brand' - The brand of the model.
+ *  - 'model' - The name of the model.
+ *  - 'version' - The version of the build, defaults to latest.
+ * @returns {String} The URL of the specified initials texture.
+ */
+ripe.Ripe.prototype.getFontUrl = function(name, format, options) {
+    options = this._getFontOptions(options);
+    format = format || "ttf";

```

Comment:
I'd simplify like this... what do you think?

```suggestion
ripe.Ripe.prototype.getFontUrl = function(name, format = "ttf", options) {
    options = this._getFontOptions(options);
```

Reply:
I usually prefer to have as it is, becase you can do 
`getFontUrl ("Example", undefined, { stuff here })`
and it will always preserve the default, otherwise you would have to spcify the default

Reply:
But I can change if you think I should

Reply:
it was only a matter of simplification but indeed there's an advantage for that usecase